.github/workflows/workflow.yml

```diff
@@ -8,6 +8,9 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
```

Comment:
were you seeing an error that required adding this?

Reply:
yes, there was an error during build which said unsupported engine =

Reply:
we should be modifying this in `action.yml` instead, so that users of the action don't need to set this

Reply:
We have use node 16 in the action.yml but our cli and workflow have a mismatch, should we not change that here?

Reply:
This is the same workflow file that users will need to setup to use our action. If we're running into an error using the action with our cli, then we should update either the cli or the action. If we can't update the versions to match, and we need to make a change here then we should update the documentation in the readme so that users know to make the same change

Reply:
This may just require updating `- uses: actions/checkout@v2` to `- uses: actions/checkout@v3` in our workflow and the example in the readme

Reply:
I am going to try running it one more time to ensure it works and its just a warning and still functions as required